### DISCUSSION TITLE
Multiple libs use the same git tags

### DISCUSSION THREAD

**Discussion by shaharkazaz** [jscutlery/semver#285] [Q&A] [answered]
Hi!

I've recently migrated my project to NX, and I have several libs that I release independently.
They were all managed under the same changelog, and the git tags represented only the version of the "main" lib. (there are several plugins to the main lib)
Now I'm trying to work with this plugin, but I see when I dry-run the bump that all the libs are taking their version from the git tag, which only represents the "main" lib in its current state.

Any idea how do I align everything to work as expected?

**Comment by edbzn** [comment, marked as answer]
Hi,

Actually, semver use the Git tags to bump a library (eg: analyzing commits from the previous version to define the new version), regardless of using sync or independent mode. The package.json "version" property is not used to read the actual version, this means that you need to tag your libraries to work with semver.

eg: 

```text
main-0.0.1
plugin-a-0.0.1
plugin-b-0.0.1
...
```

Maybe the sync mode would be better for your use case? Let me know if I can help you more.

**Reply by shaharkazaz** [reply]
I don't think that sync mode is what I need. I want to be able to release the packages independently.
I'll create a new tag for each plugin 👍
